zfsbootmenu/lib/zfsbootmenu-lib.sh

```diff
@@ -206,14 +253,24 @@ draw_snapshots() {
 
   zdebug "snapshots: ${snapshots[*]}"
 
-  if ! selected="$( HELP_SECTION=snapshot-management ${FUZZYSEL} \
+  local LEGACY_CONTEXT
+  if [ -z "${HAS_BORDER_LABEL}" ]; then
+    LEGACY_CONTEXT=1
+  fi
```

Comment:
Maybe call this `PREVIEW_PAD` or something more obvious, and set it to 0 or 1 explicitly to avoid the `${LEGACY_CONTEXT:-0}` substitution below.

Reply:
`LEGACY_CONTEXT` does double duty here. It's a flag to indicate that an optional parameter should be passed to `zfsbootmenu-preview`, and it's abused to expand the preview window height by one.  It's not a very pretty construct, so maybe a comment indicating what it's doing is in order?